File: coaching_academy/lib/MVVM/models/WorkingHour.dart
class WorkingHour {
  final String startHour;
  final String endHour;
  final String trainingOption;

  WorkingHour({
    required this.startHour,
    required this.endHour,
    required this.trainingOption,
  });

  factory WorkingHour.fromJson(Map<String, dynamic> json) {
    return WorkingHour(
      startHour: json['start_hour'],
      endHour: json['end_hour'],
      trainingOption: json['training_option'],
    );
  }
}

File: coaching_academy_api/app/Models/WorkingHour.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use App\Models\User;
use App\Models\Service;

class WorkingHour extends Model
{
    use HasFactory;
    protected $fillable = [
        "service_id",
        'start_hour',
        'end_hour',
        'training_option',
    ];

    public function service()
    {
        return $this->belongsTo(Service::class);
    }


   /*  public function user()
    {
        return $this->belongsTo(User::class);
    } */
}
